### PR TITLE
SuperAdmin entity

### PR DESCRIPTION
**Pull request by Milicerro**
I worked on the model, controller, validations and routes of Super Admin entity. I have tested the differents endpoints in postman.

### CODE REVIEW

File: src/controllers/super-admin.js [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+    });
+
+    return res.status(201).json({
+      message: 'Super admin was succesfully created',
+      data: newSuperAdmin,
+      error: false
+    });
+  } catch (error) {
+    res.status(500).json({
+      message: 'An Error ocurred',
```
Comment (Tilajc) [inline src/controllers/super-admin.js:78]:
```suggestion
      message: `An error ocurred: ${error}`,
```

File: src/controllers/super-admin.js [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+        error: true
+      });
+    }
+
+    return res.status(200).json({
+      message: `Super admin with id: ${id} was successfully deleted`
+    });
+  } catch (error) {
+    res.status(500).json({
+      message: 'An error occurred',
```
Comment (Tilajc) [inline src/controllers/super-admin.js:190]:
```suggestion
      message: `An error ocurred: ${error}`,
```

**Review by Tilajc — CHANGES_REQUESTED** [review]
It's almost done! just two comments for consistence. Pretty good job!

**Comment by LucianoStradiot** [comment]
[Super-admin entity](https://trello.com/c/bQRLx0rv/30-super-admin-entity)

**Review by LucianoStradiot — APPROVED** [review]
LG!

**Review by Tilajc — APPROVED** [review]

**Pull request merged**